src/node/tests/elder.rs

```diff
@@ -633,6 +633,7 @@ fn handle_bounced_unknown_message() {
 }
 
 #[test]
+#[ignore] //FIXME Any message invalidly signed will not deserialise / be created
```

Comment:
This test isn't about invalidly signed messages, but about messages with untrusted proof which we should handle via the bounce thing.

Reply:
Yes IIRC it was the untrusted proof signature failure here. Not the src beign recodnised, just the sig failed.

Reply:
Ah, this is the mysterious failure you wanted me to investigate? I'll look into it.